conf/logback.xml

```diff
@@ -109,6 +125,6 @@
   </logger-->
 
   <root level="INFO">
-    <appender-ref ref="CONSOLE" />
```

Comment:
Dynamically controlling the config file seems like a complicated way of customizing the config file. It would be better to use `logback. configurationFile` system property to specify a different config file, rather than make this reference one more complicated.